packages/compass-components/src/index.ts

```diff
@@ -34,6 +34,9 @@ export {
 export { Checkbox } from './components/checkbox';
 export { default as LeafyGreenProvider } from '@leafygreen-ui/leafygreen-provider';
 
+import Box from '@leafygreen-ui/box';
+export { Box };
```

Comment:
This one is really low level, so I wanted to ask if this is something we are planning to do more now, using Box instead of just html elements?

Reply:
Not sure. They kind of use it as a link here in example: https://www.mongodb.design/component/box/example/
When a link from typography has a very specific style with an icon: https://www.mongodb.design/component/typography/example/
I am ok with having just `<a></a>` here, but I thought the idea was to use leafy green as much as possible.

Reply:
Actionable elements on the screen should be buttons or links otherwise they can't be accessed by keyboard. If you need it to look like a link, you can check out how we do this in other places:

https://github.com/mongodb-js/compass/blob/b06c5ed05524d60d4705c7bea0df33b018a7342a/packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-actions.tsx#L55-L66

Reply:
Thx for sharing the example! Will do this change!